sandboxed-fs.js

```diff
@@ -4,11 +4,109 @@ const sandboxedFs = {};
 module.exports = sandboxedFs;
 
 const fs = require('fs');
+const pathModule = require('path');
+
+const errorMessage = 'path must be a string';
```

Comment:
Nit: the common Node convention is to start error messages with a capital letter.